python/lsst/ts/cbp/csc.py

```diff
@@ -81,10 +81,14 @@ async def do_moveAzimuth(self, data):
         None
 
         """
-        self.log.debug("Begin moveAzimuth")
-        self.assert_enabled("moveAzimuth")
-        self.model.move_azimuth(data.azimuth)
-        self.log.debug("moveAzimuth sent to model")
+        self.log.debug("Begin move")
+        self.assert_enabled("move")
+        await asyncio.join(
+            [
+                self.model.move_altitude(data.altitude),
+                self.model.move_azimuth(data.azimuth),
+            ]
+        )
```

Comment:
I *strongly* suggest adding a `target` event to report the commanded position. Output it whenever you get a `move` command if the command is accepted (e.g. the CBP state allows motion and the requested position is in range).

Please report it before you start the motion -- so everybody knows where the CBP is headed. Unfortunately that sounds a bit tricky with your current API because you won't know if the axes are in bounds until you start the `move_x` commands, and you wait for them to finish.

Please add such events for **all** commands that move the CBP: set mask, change mask rotation angle, set filter, park, unpark, etc.: report the new commanded state when you start making it happen. Also report the actual state, so users know when the commanded action succeeded, or failed.

Reply:
Good idea.